.clang-format

```diff
@@ -1,9 +1,7 @@
-BasedOnStyle: WebKit
+BasedOnStyle: Google
```

Comment:
I've made these changes cause origin version of the clang-format file doesn't format code as we have in existing proto files.